blobmodel/model.py

```diff
@@ -81,7 +82,9 @@ def __init__(
             T=T,
             periodic_y=periodic_y,
         )
-        self.blob_shape: str = blob_shape
+        self.blob_shape = (
```

Comment:
Ah OK, I guess this handles it my previous point. In this case I would still keep strings in the examples since it is easier to use (one less import)